source/include/dqm4hep/Internal.h

```diff
@@ -359,6 +360,43 @@ namespace dqm4hep {
 
       return true;
     }
+    
+    //-------------------------------------------------------------------------------------------------
+
+    /**
+     *  @brief  Get the current process id
+     */
+    inline int pid() {
+#if _WIN32
+      return GetCurrentProcessId();
+#else
+      return ::getpid();
```

Comment:
I was skeptical but this actually works on APPLE too.

Reply:
Nice to know !